### DISCUSSION TITLE
Create new coil

### DISCUSSION THREAD

**Discussion by FrancescoLomi** [simnibs/simnibs#367] [8. Simulation workflows]
Hi everybody!
I'm trying to create the coil model for simnibs using the script of the last version, but I encounter this error related to the scipy package, although I've reinstalled it (1.13.1 version). 
If you had suggestions they would be highly appreciated.
Thanks you so much!
![Immagine 2024-06-17 123440](https://github.com/simnibs/simnibs/assets/118944911/5755ff93-6f0f-4bde-a13b-d4069b024266)

**Comment by bjornsig** [comment]
Can you try to uninstall all simnibs versions you have on your computer and reinstall the newest version?
